CHANGES.rst

```diff
@@ -51,6 +51,9 @@ Release History
   the Gram system.
   (`#1027 <https://github.com/nengo/nengo/pull/1027>`_,
   `#1019 <https://github.com/nengo/nengo/issues/1019>`_)
+- Allow recurrence of the form ``a=b, b=a`` in basal ganglia SPA actions.
+  (`#1098 <https://github.com/nengo/nengo/issues/1098>`_,
+   `#1099 <https://github.com/nengo/nengo/pull/1099>`_)
```

Comment:
Extra space at the start of this line is causing Github to [render weirdly](https://github.com/nengo/nengo/blob/61cb06007a8063e1ddbd115373354dbd0c56f0a5/CHANGES.rst). Will fix in the merge if you don't get to it!